File: google-auth/src/main/java/com/kevalpatel2106/googleauth/GoogleAuthResponse.java
package com.kevalpatel2106.googleauth;

/**
 * Created by multidots on 6/21/2016.
 */

public interface GoogleAuthResponse {

    void onGoogleAuthSignIn(GoogleAuthUser user);

    void onGoogleAuthSignInFailed();

    void onGoogleAuthSignOut(boolean isSuccess);
}

File: google-auth/src/main/java/com/kevalpatel2106/googleauth/GoogleAuthResponse.kt
package com.kevalpatel2106.googleauth

/**
 * Created by Keval on 6/21/2016.
 */

interface GoogleAuthResponse {

    fun onGoogleAuthSignIn(user: GoogleAuthUser)

    fun onGoogleAuthSignInFailed()

    fun onGoogleAuthSignOut(isSuccess: Boolean)
}
